tests/acceptance/settings-profile-test.js

```diff
@@ -99,12 +100,12 @@ test("it allows editing of users image", (assert) => {
 
   andThen(() => {
     fillInFileInput('.image-drop input', { name: fileName, content: droppedImageString });
-    click('.save');
+    settingsProfilePage.userSettingsForm.clickSave();
   });
 
   andThen(() => {
-    assert.equal(find('.alert-success').length, 1);
-    assert.equal(find('.alert-success p').text(), "Profile updated successfully");
+    assert.ok(settingsProfilePage.successAlert.isVisible);
+    assert.equal(settingsProfilePage.successAlert.message, "Profile updated successfully");
     let expectedStyle = `url(${droppedImageString})`;
     assert.equal(removeDoubleQuotes(find('.image-drop').css('background-image')), expectedStyle);
```

Comment:
Don't forget to get rid of unused imports from the test module if you end up going in this direction.